### DISCUSSION TITLE
Gameplay offset display pops up when adjusting offset during replay playback

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#26144] [Q&A] [answered]
https://github.com/ppy/osu/assets/53081750/a0fac4b6-faa0-4fdc-9d1c-5ee79ac05089

**Comment by peppy** [comment]
This really just needs to be a different kind of display. Because imagine the case where a user is adjusting via hotkeys from a replay context. You'd still want this to display.

I'll remove it for now but it really does need future consideration.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/26543/files
